### PR TITLE
added template for landing page and home page

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
1. Created template for landing page and home page
2. Removed unwanted files

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,17 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Routes, Route } from "react-router-dom";
+import { LandingPage } from './Pages';
+import { Home} from "./Pages/index";
+import { Navbar } from "./component/index";
```
Comment (ravikumar0403) [inline src/App.js]: You have added the config for path alias so you can omit `./` from the imports

File: src/App.js [hunk 2]
```diff
@@ -1,41 +1,17 @@
...
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Navbar />
+      <Routes>
+        <Route path='/' element={<LandingPage />}></Route>
+        <Route path='/home' element={<Home />}></Route>
```
Comment (ravikumar0403) [inline src/App.js]: This can be a self closing tag as there are no children for these components

File: src/Pages/home-page/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import './home.css';
+import { Drawer, SearchBar, Card } from "component/index";
```
Comment (ravikumar0403) [inline src/Pages/home-page/Home.jsx]: `/index` can be omitted

File: src/Pages/home-page/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    <div className="wrapper">
+      <div className="container d-flex">
+        <Drawer />
+        <div className="d-flex justify-center center-body">
+          <div class="drawer-app-content">
+            <header class="drawer-top-bar">
+              <SearchBar />
+            </header>
+            <main class="home-page-body">
+              <div class="fixed"></div>
```
Comment (ravikumar0403) [inline src/Pages/home-page/Home.jsx]: you have missed changing class to className here

File: src/component/drawer/Drawer.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import './drawer.css';
+import HomeOutlinedIcon from '@mui/icons-material/HomeOutlined';
+import LabelOutlinedIcon from '@mui/icons-material/LabelOutlined';
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined';
+import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
+import FaceOutlinedIcon from '@mui/icons-material/FaceOutlined';
```
Comment (ravikumar0403) [inline src/component/drawer/Drawer.jsx:6]: MUI provides named imports as well, so you can use named imports to make your code cleaner

File: src/component/drawer/Drawer.jsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+            </span>
+          </li>
+          <li class="nav-item d-flex items-center py-2 px-2">
+            <span class="d-flex items-center">
+              <DeleteOutlineOutlinedIcon className="mr-2 icon" />
+              Trash
+            </span>
+          </li>
+          <li class="nav-item d-flex items-center py-2 px-2">
+            <span class="d-flex items-center">
```
Comment (ravikumar0403) [inline src/component/drawer/Drawer.jsx]: missed changing class to className in this file as well

File: src/component/drawer/drawer.css [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  top: 0px;
+  right: 0px;
+  left: 0px;
+  padding: 1.12rem;
+}
+.home-page-body {
+  height: 100%;
+  overflow-y: auto;
+  padding: 0px 18px;
+  /* gap: 1rem; */
```
Comment (ravikumar0403) [inline src/component/drawer/drawer.css:51]: remove commented code if not required

**Review by ravikumar0403 — APPROVED** [review]
Good folder structure and code is also clean. 
Remove the commented code if not required and you can do a global search to replace class with className

File: src/component/navbar/Navbar.jsx [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+                        <rect y="20" width="60" height="10"></rect>
+                        <rect y="40" width="60" height="10"></rect>
+                    </svg>
+                    {/* <div className="logo-wrapper"><img src={logo} className="responsive-img"/></div> */}
+                    <Link className="brand-logo mx-2" to="/">Noter</Link>
+                    {/* <ul className="navbar-nav navbar-nav-collapse">
+                        <li className="nav-item">
+                            <Link to='/' className='nav-item-link'>HOME</Link>
+                        </li>
+                        <li className="nav-item">
```
Comment (sakshi006) [inline src/component/navbar/Navbar.jsx:27]: You can remove these commented codes

**Review by sakshi006 — APPROVED** [review]
Looks good to merge
I liked the folder structure and naming conventions.

File: src/App.js (hunk 2 above)
Comment (SHIKHASINGH1506) [reply src/App.js]: Will change it.

File: src/component/drawer/Drawer.jsx (hunk 5 above)
Comment (SHIKHASINGH1506) [reply src/component/drawer/Drawer.jsx:6]: read about it and changed.

File: src/App.js (hunk 1 above)
Comment (SHIKHASINGH1506) [reply src/App.js]: Actually I added config path after these imports and forgot to update it later. Will change it now.

**Pull request merged**